bedrock/firefox/templates/firefox/all.html

```diff
@@ -30,7 +30,7 @@
   sky firefox-all-{{ channel }} {% if channel == 'alpha' %}blueprint{% endif %}
 {% endblock %}
 
-{% block site_css %}
+{% block page_css %}
```

Comment:
>> On a related note, moving all these pages from site_css to page_css will result in 1 extra HTTP request per page, as we're now loading two separate bundles. Are we okay adding this overhead?

> I think we need to weigh the potential cost of the extra request against the potential savings of smaller files, especially if the global site styles are cached from a previous page. I'd love to do some performance testing to compare load times.

+1, I've been guilty of [obsessing over HTTP request reduction](http://blog.getify.com/obsessions-http-request-reduction/) myself, but sometimes splitting the files up can yield better performance, despite the request overhead.